app/queries/display_ads/filtered_ads_query.rb

```diff
@@ -56,6 +59,15 @@ def authenticated_ads(display_auth_audience)
       @filtered_display_ads.where(display_to: display_auth_audience)
     end
 
+    def community_or_in_house_ads
+      @filtered_display_ads.where(
+        "(type_of = :in_house) OR
+         (type_of = :community AND organization_id = :organization_id) OR
+	       (type_of = :external AND organization_id != :organization_id)",
+          DisplayAd.type_ofs.merge({ organization_id: @organization_id }),
```

Comment:
We probably want to have some test coverage here, showing that the filters work as expected.